src/conv/k-g/extrude.cpp

```diff
@@ -0,0 +1,90 @@
+/*                 E X T R U D E . C P P
+ * BRL-CAD
+ *
+ * Copyright (c) 2024 United States Government as represented by
+ * the U.S. Army Research Laboratory.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this file; see the file named COPYING for more
+ * information.
+ */
+ /** @file extrude.cpp
+  *
+  * LS Dyna keyword file to BRL-CAD converter:
+  * intermediate extrude implementation
+  */
+
+#include "extrude.h"
+
+
+
+
+Extrude::Extrude(void)
+{
+    BU_GET(m_extrude, rt_extrude_internal);
+    m_extrude->magic = RT_EXTRUDE_INTERNAL_MAGIC;
+}
+
+void Extrude::setName
+(
+    const char* value
+) {
+    if (value != nullptr)
+	name = value;
+    else
+	name = "";
+}
+
+const char* Extrude::getName(void) const
+{
+    return nullptr;
```

Comment:
That's an unexpected return value. Same for Sketch.
However, getName() seems to be never used from any primitive. They could all be removed?

Reply:
Yes, I think I just added the functions and forgot to implement them. also you are right these functions are not uesed, I will delet them